lms/djangoapps/support/static/support/jsx/entitlements/entitlements.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+export class EntitlementSupportPage extends React.Component {
+  constructor(props){
+    super(props)
+  }
+  render(){
+    return	(
+      <div>
+        Base Entitlement Support Page
+      </div>
+    )
+  }
+}
```

Comment:
Sane spacing @Mjloturco !

Reply:
Yeah, looks like 2 space tabs in sublime weren't playing well with github.